book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
```

Comment:
- [ ] Please add a method for adding a rental as Book and Person class has-many-through relationship using the Rental class. The book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship